### PR TITLE
Recipe for swat

### PR DESCRIPTION
**Pull request by kesmit13**
Initial recipe for SAS SWAT module

### CODE REVIEW

File: recipes/swat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+requirements:
+    build:
+        - python
+        - pip
+        - setuptools
+        - pandas
+        - six
+        - requests
+        - libstdcxx-ng  # [linux64]
```
Comment (mariusvniekerk) [inline recipes/swat/meta.yaml]: Why are these needed?  It doesn't seem as if the source on github has any c things around

File: recipes/swat/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        - libgcc-ng  # [linux64]
+        - libuuid  # [linux64]
+
+test:
+    imports:
+        - swat
+
+#outputs:
```
Comment (mariusvniekerk) [inline recipes/swat/meta.yaml]: you probably want to remove this

File: recipes/swat/meta.yaml (hunk 1 above)
Comment (mariusvniekerk) [reply recipes/swat/meta.yaml]: I'd recommend that you add ` - compiler('cxx') ` here instead of having libgcc and libstdcxx.  
If this library ships precompiled binaries in its tarball releases this may be problematic when we switch compilers.

Comment (kesmit13) [reply recipes/swat/meta.yaml]: We do use pre-compiled binaries that are distributed in the tarball.  I don't know that the C dependencies are really needed since these are part of the manylinux1 spec; this package should run on most Linux distributions.  I just added them because conda build suggested it when I was writing the recipe.  I can change it to compiler('cxx') (or maybe just remove them?).

Comment (marcelotrevisani) [reply recipes/swat/meta.yaml]: I think you can just remove that and also from run, if you really need that please add

```
  - {{ compiler('cxx') }}
  - {{ compiler('c') }}
```

into the ``build`` requirements

Comment (marcelotrevisani) [reply recipes/swat/meta.yaml]: I think ``setuptools`` will not be necessary as well. Could you please try to remove it?

**Comment by kesmit13** [comment]
@conda-forge/staged-recipes I believe my recipe is all in order now.  I've addressed all of the suggestions and it's passing the checks.  I was just wondering if there is anything else I need to do?

**Comment by marcelotrevisani** [comment]
LGTM

**Comment by marcelotrevisani** [comment]
Wait.

There is some things to fix in my opinion:

```
WARNING (swat,lib/python3.6/site-packages/swat/lib/linux/t0d0en.so): Needed DSO lib/libgcc_s.so.1 found in ['libgcc-ng']
WARNING (swat,lib/python3.6/site-packages/swat/lib/linux/t0d0en.so): .. but ['libgcc-ng'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
WARNING (swat,lib/python3.6/site-packages/swat/lib/linux/t0d0en.so): did not find - or even know where to look for: /lib64/libcrypt.so.1
WARNING (swat,lib/python3.6/site-packages/swat/lib/linux/t0d0en.so): did not find - or even know where to look for: /lib64/libpthread.so.0
WARNING (swat,lib/python3.6/site-packages/swat/lib/linux/t0d0en.so): did not find - or even know where to look for: /lib64/libm.so.6
WARNING (swat,lib/python3.6/site-packages/swat/lib/linux/t0d0en.so): did not find - or even know where to look for: /lib64/libc.so.6
WARNING (swat,lib/python3.6/site-packages/swat/lib/linux/tk4acons.so): did not find - or even know where to look for: /usr/lib64/libstdc++.so.6
WARNING (swat,lib/python3.6/site-packages/swat/lib/linux/tk4acons.so): did not find - or even know where to look for: /lib64/libdl.so.2
```

**Comment by kesmit13** [comment]
I had libgcc-ng and compiler('cxx') in the recipe at different times, but the suggestion was to remove them.  As for the other libraries, those are all in the linuxmany1 list of libraries, so I would imagine most linux distributions have them.

**Comment by kesmit13** [comment]
I'm still not sure what needs to be done to make these other warnings go away.  When I do test builds locally conda build asks if patchelf should be run, but it was.  So I'm not sure what magic needs to be done to the recipe to make it work.

**Comment by kesmit13** [comment]
Here is the line from the log that runs patchelf.

```
patchelf: file: /r/ge.unx.sas.com/vol/vol510/u51/kesmit/conda-bld/swat_1529070669784/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.5/site-packages/swat/lib/linux/tkcop.so
    setting rpath to: $ORIGIN/../../../../..:$ORIGIN/../../../../../linux:/lib64/:/usr/lib64/
```

But here is the warning that I still get.

```
WARNING (swat,lib/python3.5/site-packages/swat/lib/linux/tkcop.so): /lib64/libpthread.so.0 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?

**Comment by kesmit13** [comment]
I think since we might try to take a different direction with this package and put it on Anaconda Cloud in an organization account.  I'll close it, and re-open later if needed.
